File: demos/rubymotion/app/models/post.rb
class Post < NSRRemoteObject
  attr_accessor :author, :content, :responses, :created_at
  
  def remoteProperties
    super + ["author", "content", "responses", "created_at"]
  end
  
  def nestedClassForProperty(property)
    Response if property == "responses"
  end
end

File: demos/swift/SwiftDemo/Post.swift
//
//  Post.swift
//  SwiftDemo
//
//  Created by Dan Hassin on 8/21/14.
//  Copyright (c) 2014 Dan Hassin. All rights reserved.
//

import UIKit

@objc(Post) class Post: NSRRemoteObject {
    var content:String!
    var author:String!
    var responses:[Response]!
    var createdAt:NSDate!
    
    override func nestedClassForProperty(property: String!) -> AnyClass! {
        if property == "responses" {
            return Response.self
        }
        
        return super.nestedClassForProperty(property)
    }
}
